package.json

```diff
@@ -144,7 +144,7 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.20.5",
     "eslint-plugin-react-hooks": "^4.0.8",
-    "firebase-admin": "^9.1.1",
+    "firebase-admin": "^9.4.1",
```

Comment:
Are these changes required for this feature specifically? Or just 'came along' from your local changes when you opened the PR? If the latter, it would be good to get the changes here + in `package-lock.json` cleaned up/reverted for this PR.